src/routes/bitbucket/proxy/index.ts

```diff
@@ -42,6 +42,17 @@ export function proxyRoutes(runner: Runner, client: BitbucketClient) {
       ]);
 
       if (permission(permissionLevel).isAtLeast('land')) {
+        if (requestStatus?.state === 'merging' || requestStatus?.state === 'success') {
```

Comment:
nit: this could be nicer syntax.
```suggestion
        if (['merging', 'success'].includes(requestStatus?.state) {
```

Also isn't github pr suggestions just the greatest 🤤

Reply:
It is until you realize that you applied the suggestion before checking if it had a syntax error 😓

Reply:
sorry.. waiting for github pr view to become a fully fledged editor with syntax highlighting